libs/api/domains/hms-loans/src/lib/dto/getHmsLoansPaymenthistory.input.ts

```diff
@@ -0,0 +1,9 @@
+import { Field, InputType } from '@nestjs/graphql'
+import { IsNumber } from 'class-validator'
+
+@InputType()
+export class GetHmsLoansPaymenthistoryInput {
```

Comment:
```suggestion
export class GetHmsLoansPaymentHistoryInput {
```

the h is annoyingly lower case